<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mountOptions</class>
 <widget class="QDialog" name="mountOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>113</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>file system options</string>
  </property>
  <widget class="QPushButton" name="pbOk">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Ok</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>481</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter Comma Separated Volume's File System Options Below</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditOptions">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditOptions</tabstop>
  <tabstop>pbOk</tabstop>
  <tabstop>pbCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
